charts/flux2/templates/kustomize-controller.yaml

```diff
@@ -36,6 +36,17 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- if .Values.kustomizecontroller.envFrom.enabled }}
```

Comment:
That can be replaced by:
{{- if or (.Values.kustomizecontroller.envFrom.map.name) (.Values.kustomizecontroller.envFrom.secret.name) }}